java/ql/lib/semmle/code/java/frameworks/Jdbc.qll

```diff
@@ -2,7 +2,7 @@
  * Provides classes and predicates for working with the Java JDBC API.
  */
 
-private import semmle.code.java.dataflow.ExternalFlow
+private import java
```

Comment:
NIT: I guess these `private import java` don't need to be `private` (although it doesn't hurt...)

Reply:
Yes, I just didn't want to re-expose something that was not exposed before as this is a library, but I will change it for consistency.